daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2471,6 +2472,7 @@ func (devices *DeviceSet) Status() *Status {
 		}
 	}
 
+	status.MinFreeSpace = fmt.Sprintf("%v%%", devices.minFreeSpacePercent)
```

Comment:
Actually, in the API, we shouldn't do any formatting, so either raw percentage, or calculated size